core/arch/arm/plat-rpi3/main.c

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (c) 2016, Sequitur Labs Inc. All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
```

Comment:
I don't know the answer whether it was intentional or not. But, it should be on the same format as the rest (which is how the BSD 2-Clause is written). I'll change it.